Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/MigrationTests/SpannerMigrationTest.cs

```diff
@@ -446,7 +446,7 @@ public void ShouldThrowLengthValidationException()
                 "this is too long string should throw length validation error"
             });
 
-            Assert.Throws<SpannerBatchNonQueryException>(() => context.SaveChanges());
+            Assert.Throws<SpannerException>(() => context.SaveChanges());
```

Comment:
This error has changed, because implicit transactions no longer use Batch DML. Only Batch DML will throw `SpannerBatchNonQueryException`.